packages/carousel/src/react/__specs__/index.spec.tsx

```diff
@@ -30,16 +30,22 @@ describe('Carousel', () => {
     })
 
     it('should hide the prev control', () => {
-      const control = container
-        .querySelector('[aria-label="get previous carousel page"]')
-        .closest('[data-testid="carousel control"]')
+      const page = container.querySelector(
+        '[aria-label="get previous carousel page"]'
+      )
+      expect(page).toBeDefined()
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+      const control = page!.closest('[data-testid="carousel control"]')
```

Comment:
I like the assertion here.  I don't like that I have to disable eslint.  The alternative is to put an `if` block around this.

Reply:
maybe disable this rule for .spec files 🤷